cmd/spire-server/cli/run/run.go

```diff
@@ -357,6 +359,12 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 	}
 	sc.Log = logger
 
+	if c.Server.AttestLimit <= 0 {
```

Comment:
I don't think that we need this. If `attest_limit` is not set or is set to 0, it will get the default value of 1 and this condition will never be true.

Reply:
Right, perhaps we can keep just it in case someone provides a negative number as attestation limit?

```suggestion
	if c.Server.AttestLimit < 0 {
```

Reply:
Oh, right. Yes, I think that we need to keep the validation to check that only a positive value is set.